File: Solutions/1-9/9/Nine.java
/**
 * Solves Project Euler problem #9
 * https://projecteuler.net/problem=9
 * 
 * @author Jake Conway
 * @version 1.0
 */
public class Nine
{
    public static void main(String[] args)
    {
        for(int i = 1; i <= 1000; i++) 
        {
            for(int j = 1; j <=i+1; j++)
            {
                int a = i;
                int b = j;
                int c = 1000-(a+b);
                if((Math.pow(a, 2) + Math.pow(b, 2)) == Math.pow(c, 2))
                {
                    System.out.println("Solution: " + (a*b*c));
                }
            }
        }
    }
}

File: Solutions/1-9/9/nine.py
class Nine:
	@staticmethod
	def main():
		for i in range(1,1000):
			for j in range(1, i+1):
				c = 1000 - (i+j)
				if i**2 + j**2 == c**2:
					print("Solution: " + str(i*j*c))
					
if __name__ == '__main__':
	Nine.main()
